src/main/java/seedu/address/model/person/exceptions/PersonWrongType.java

```diff
@@ -0,0 +1,6 @@
+package seedu.address.model.person.exceptions;
+
+/**
+ * Signals that the operation is unable to act on given type of person
+ */
+public class PersonWrongType extends Exception {}
```

Comment:
Wrong format for exceptions. Exceptions should be clear that they are exceptions, and thus the recommended naming syntax could be `PersonWrongTypeException`